api/api/utils/image_proxy/__init__.py

```diff
@@ -185,16 +185,15 @@ async def get(
                     "occurrences", settings.THUMBNAIL_ERROR_REPEATED_ALERT_FREQUENCY
                 )
                 sentry_sdk.capture_exception(exc)
-        if isinstance(exc, requests.exceptions.HTTPError):
-            code = exc.response.status_code
+        if isinstance(exc, ClientResponseError):
+            status = exc.status
             do_not_wait_for(
-                tallies_incr(
-                    f"thumbnail_http_error:{domain}:{month}:{code}:{exc.response.text}"
-                )
+                tallies_incr(f"thumbnail_http_error:{domain}:{month}:{status}")
```

Comment:
The response text is ambiguous from aiohttp (I wasn't entirely sure what it _was_ and whether it matched wherever requests as using). I removed it because I figure we really care more about the _status_ code, and if we need specific details about the error itself, we can look at the logs for the exc.message or review the captured exception in sentry (if relevant).